pyproject.toml

```diff
@@ -15,7 +15,7 @@ keywords = ["magic link", "authentication", "passwordless"]
 [tool.poetry.dependencies]
 python = "^3.6"
 Django = ">=2.1"
-packaging = ">=20.9,<22.0"
+packaging = ">=20.9,<24.0"
```

Comment:
```suggestion
packaging = ">=20.9"
```

Reply:
Hey @Mogost , I am not going to change my PR. 

I personally just want this package to be updated. I am just as happy if Matt @pyepye uses this PR or your PR at #30 to solve #27 

Thanks for this.